torch/csrc/jit/script/compiler.cpp

```diff
@@ -43,6 +43,7 @@ struct PrintValue : public SugaredValue {
     Method & m,
     at::ArrayRef<NamedValue> inputs,
     at::ArrayRef<NamedValue> attributes,
+    const std::vector<std::shared_ptr<SugaredValue>> &blocks,
```

Comment:
I don't like this because Fork is going to be the _only_ thing ever uses blocks.